src/pages/in/embed/_id/list.vue

```diff
@@ -134,16 +137,26 @@ export default {
   },
   async mounted() {
     const { params, query } = this.$route;
+    const iscnId = query.iscn_id;
     const referrer = handleQueryStringInUrl(query.referrer);
-    const promises = [
-      apiGetLikeButtonLikerList(params.id, referrer),
-      apiGetLikeButtonTotalCount(params.id, referrer),
-    ];
-    if (referrer) {
-      const url = encodeURI(referrer);
-      /* Try to get html to fetch title below */
-      if (checkValidDomainNotIP(url)) {
-        promises.push(apiGetPageTitle(referrer));
+    let promises;
+    if (iscnId) {
+      promises = [
+        apiGetLikeButtonLikerListByIscnId(iscnId),
+        apiGetLikeButtonTotalCountByIscnId(iscnId),
+      ];
+      promises.push();
```

Comment:
Why push empty?
Should push above?
Can just `push()` promise instead of reassignment
```suggestion
      promises.push(
        apiGetLikeButtonLikerListByIscnId(iscnId),
        apiGetLikeButtonTotalCountByIscnId(iscnId),
      );
```